client/src/Tickets/Tickets.jsx

```diff
@@ -103,15 +87,16 @@ const Tickets = () => {
         });
         setData(data);
       });
-    if (user) {
-      setTicket((prevTicket) => ({ ...prevTicket, UserId: user.id }));
```

Comment:
Don't do things like this! Notice how I updated the SERVER api route to set the user id based on who's calling the API... so this will be done automatically by the server (and enforced by session authorization).